File: documentation/user/en/query/requirements/examples/facet/facet-summary-of-reference-filtering.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEquals("status", "ACTIVE")
        	),
        	Require(
        		FacetSummaryOfReference(
        			"parameterValues",
        			Counts,
        			FilterBy(
        				AttributeContains("code", "ar")
        			),
        			FilterGroupBy(
        				AttributeStartsWith("code", "o")
        			),
        			EntityFetch(
        				AttributeContent("code")
        			),
        			EntityGroupFetch(
        				AttributeContent("code")
        			)
        		)
        	)
        )
	)
);

File: documentation/user/en/query/requirements/examples/facet/facet-summary-of-reference-filtering.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEquals("status", "ACTIVE")
				),
				require(
					facetSummaryOfReference(
						"parameterValues",
						filterBy(
							attributeContains("code", "ar")
						),
						filterGroupBy(
							attributeStartsWith("code", "o")
						),
						entityFetch(
							attributeContent("code")
						),
						entityGroupFetch(
							attributeContent("code")
						)
					)
				)
			)
		);
	}
);